internal/updater/automatic.go

```diff
@@ -26,3 +26,8 @@ func Enable(a fyne.App, w fyne.Window) {
 		return
 	}
 }
+
+// TODO: Disable turns off automatic application updates.
+//func Disable() {
+//
+//}
```

Comment:
See https://github.com/Jacalz/rymdport/pull/77#discussion_r1143707594. I think you can remove this entirely.

Reply:
got it